package.json

```diff
@@ -49,6 +49,17 @@
                 }
             ]
         },
+        "configuration": {
+            "title": "JFrog",
+            "properties": {
+                "jfrog.xray.exclusions": {
+                    "type": "string",
+                    "default": "**/*{test,venv,node_modules}*",
+                    "scope": "resource",
+                    "markdownDescription": "A [glob pattern](https://code.visualstudio.com/api/references/vscode-api#GlobPattern) that will be matched on file paths to exclude files and directories to scan."
```

Comment:
...that will be matched on file paths to exclude files and directories to scan. -->
...used to exclude specific paths from being scanned by JFrog Xray. For example, go.mod files under directories named testdata will not be scanned.